proto/net.proto

```diff
@@ -81,14 +81,45 @@ message TrainingParams {
   optional string lc0_params = 6;
 }
 
+message NetworkFormat {
+  // Format to encode the input planes with. Used by position encoder.
+  enum InputFormat {
+    INPUT_UNKNOWN = 0;
+    INPUT_CLASSICAL_112_PLANE = 1;
+  }
+  optional InputFormat input = 1;
+
+  // Output format of the NN. Used by search code to interpret results.
+  enum OutputFormat {
+    OUTPUT_UNKNOWN = 0;
+    OUTPUT_CLASSICAL = 1;
+    // OUTPUT_WDL = 2;  // Uncomment when implemented.
+  }
+  optional OutputFormat output = 2;
+
+  // Network architecture. Used by backends to build the network.
+  enum NetworkStructure {
+    NETWORK_UNKNOWN = 0;
+    NETWORK_CLASSICAL = 1;
+    // NETWORK_SE = 2;  // Uncomment when implemented.
+  }
+  optional OutputFormat network = 3;
```

Comment:
I assume this should be NetworkStructure enum.

Reply:
Done.